src/sdk/utils.ts

```diff
@@ -43,9 +46,9 @@ const setLivePreviewConfig = (): contentstack.LivePreview => {
       'Your LP config is set to true. Please make you have set all required LP config in .env'
```

Comment:
sentence has grammar error 
Your LP config is set to true. Please make sure you have set all the required LP config in .env.

Reply:
resolved